docs/ci_runner_setup/linux/docker_install.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Add Docker's official GPG key:
+sudo apt-get update
+sudo apt-get install ca-certificates curl -y
+sudo install -m 0755 -d /etc/apt/keyrings
+sudo curl -fsSL https://download.docker.com/linux/ubuntu/gpg -o /etc/apt/keyrings/docker.asc
+sudo chmod a+r /etc/apt/keyrings/docker.asc
```

Comment:
Please link to some official documentation for how this is the "official GPG key" or these are the recommended setup steps.